www/js/app/list/directive/template/movie-list.html

```diff
@@ -0,0 +1,7 @@
+<ion-item class="item-thumbnail-left" href="#/view/{{movieid}}">
```

Comment:
I totally miss that, there's absolutely no reason in using href with AngularJS, since the right way is to use ng-href.
"Using Angular markup like {{hash}} in an href attribute will make the link go to the wrong URL if the user clicks it before Angular has a chance to replace the {{hash}} markup with its value. Until Angular replaces the markup the link will be broken and will most likely return a 404 error. The ngHref directive solves this problem." (https://docs.angularjs.org/api/ng/directive/ngHref)